index.html

```diff
@@ -179,17 +180,32 @@
         loadData(uri);
         document.getElementById('uriEntered').innerHTML = "You are displaying documents in " + uri.bold();
       }
-      else { //default
+      else {
         loadData('addr.json');
       }
 
+      $("input[name='insert']").serialize(function() {
```

Comment:
I deleted that whole block of code because it isn't doing anything. Is that okay?